reccmp/ghidra_scripts/lego_util/ghidra_helper.py

```diff
@@ -109,7 +109,9 @@ def sanitize_name(name: str) -> str:
     # Importing function names like `FUN_10001234` into BETA10 can be confusing
     # because Ghidra's auto-generated functions look exactly the same.
     # Therefore, such function names are replaced by `LEGO_10001234` in the BETA10 import.
-    if GLOBALS.module == SupportedModules.BETA10:
+
+    # FIXME: The identification here is a crutch - we need a more reusable solution for this scenario
```

Comment:
This is the one place that is still isle-specific. Please tell me if you want me to find a solution before merging.

One idea I had was to add a `ghidra` section to the targets of a project configuration like so:
```yaml
  BETA10:
    filename: BETA10.DLL
    [...]
    ghidra:
      function_prefix: "LEGO1"
```

Reply:
Feel free to add whatever attributes you think are needed.
It's also be useful for a "ghidra" type in https://github.com/isledecomp/reccmp/issues/8

Reply:
What if we don't overwrite the Ghidra function name if the one from `reccmp` has the `FUN_` prefix? (Or if it matches a `temp_function_prefix` key from the yml?) We can still populate all the other information.

I don't know how common it is for a decomp to have two different versions of the same binary, but this might be an unusual situation.

Reply:
When I was working on BETA10 I found it hard to tell if a function was not matched and had Ghidra's default name, or if it was matched and was named `FUN_...` by the Ghidra script. There are also some address collisions between LEGO1  and BETA10 functions.

The "clean" solution would be to use a different pattern in the source code for unnamed functions, but that would be a major change to the existing code. It would have one additional minor benefit: It'll be easier to tell apart non-stubbed functions from stubbed / implemented ones in LEGO1.

Reply:
will be addressed in a follow-up, see #16